recipes/nine/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "nine" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3064fbeb512e756a415606a1399f49c22de867d5ac7e2b6c91c35e757d3af42d"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
can this be noarch python?

Reply:
Probably unless we care about [Python 2.6 support]( https://github.com/nandoflorestan/nine/blob/a50d0cfd8753e2e7944d2a6d7159c5f9cb3fe9ea/setup.py#L13-L16 ), which I'm guessing we don't.